lab/Makefile

```diff
@@ -53,16 +53,22 @@ lab00: stop_compose
 	cd .. && \
 	echo "!!! Don't forget add iol image into ./lab00_cisco_base/src/ios-7200 folder !!!"
 
+lab01: stop_compose
+	@cp ./topologies/lab01_frr-only-test/docker-compose.yml ./netbox-docker/docker-compose.override.yml && \
+	cd topologies/lab01_frr-only-test && \
+	cp default_configs/* operational_configs/ && \
+	cd ../../netbox-docker/ && docker-compose up -d --build && cd .. && \
+	docker exec --user root netbox-docker-netbox-1 /opt/netbox/netbox/manage.py loaddata /home/ubuntu/src/netbox-dump.json && \
```

Comment:
Maybe we can set static name for the container, instead of `netbox-docker-netbox-1`?